bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -248,6 +248,16 @@ if wflow.cp.has_option("workflow-segments", "segments-vetoes"):
                                             tags=['veto'])
     datafind_veto_files.append(veto_file)
 
+# Generate sky grid if needed
+skygrid_file = None
+if wflow.cp.has_option("workflow", "sky-error"):
+    if float(wflow.cp.get("workflow", "sky-error")):
```

Comment:
I think you can drop this line. `sky-error` may contain the unit suffix, so it would not generally convert to a float.

Reply:
But then we need `pycbc_make_sky_grid` to handle the units correctly.

Reply:
https://github.com/gwastro/pycbc/issues/4986 and then we can get back to this PR

Reply:
It looks like @titodalcanton addressed this in https://github.com/gwastro/pycbc/pull/4988/files, and also ran `black` on the executable, so I will undo my latest commit.

Reply:
Reverted: this PR does not change make_sky_grid, but it depends on https://github.com/gwastro/pycbc/pull/4988.